Software/app/basic/README.md

```diff
@@ -0,0 +1,23 @@
+# basic
+
+## Description
+
+This application is mainly used to test the hardware of the GNSE.
+
+## Configuration
+
+Some settings are available in [conf/app_conf.h](https://github.com/TheThingsIndustries/generic-node-se/tree/develop/Software/app/basic/conf/app_conf.h) to set the testing time and amount for some peripherals.
```

Comment:
Please use relative paths instead of links in all of the documentation.
Please observe the documentation in the repository as we never use links unless it's something outside of the repository.

Please apply this to all the decimation done in this pull request.

> For compatibility when deploying in genericnode.com/docs, we will provide one link and the rest of the documentation will assume using this link.

```suggestion
Some settings are available in [conf/app_conf.h](./conf/app_conf.h) to set the testing time and amount for some peripherals.
```